packages/amazonq/.changes/next-release/Bug Fix-7034adfe-fc09-44d2-8328-d908ab99f1ac.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "fix logic to allow customer to modify root source folder"
```

Comment:
I understand, the issue is that is hard to put in words because the root cause is a refactor that changed an exception that broke our logic, so basically this is fixing the logic again as said in the description.